src/node_options.cc

```diff
@@ -353,10 +353,11 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "returned)",
             &EnvironmentOptions::dns_result_order,
             kAllowedInEnvvar);
-  AddOption("--enable-network-family-autoselection",
-            "Enable network address family autodetection algorithm",
-            &EnvironmentOptions::enable_network_family_autoselection,
-            kAllowedInEnvvar);
+  AddOption("--network-family-autoselection",
+            "Disable network address family autodetection algorithm",
+            &EnvironmentOptions::network_family_autoselection,
+            kAllowedInEnvvar,
+            true);
```

Comment:
```suggestion
            true);
  Implies("--enable-network-family-autoselection", "--network-family-autoselection");
```

Reply:
If we do this, will also `--no-enable-network-family-autoselection` exist or is implicitly handled?